src/geovista/geodesic.py

```diff
@@ -164,7 +164,7 @@ def __init__(
         ----------
         lons : ArrayLike
             The longitudes (degrees) of the bounding-box, in the half-closed interval
-            ``[-180, 180)``. Note that, longitudes will be wrapped to this interval.
+            ``[-180, 180]``. Note that, longitudes will be wrapped to this interval.
```

Comment:
@trexfeathers It's a **half-open** interval, not a **closed** interval.

``[-180, 180)`` is the correct mathematical definition of the interval, see [here](https://mathworld.wolfram.com/Half-ClosedInterval.html).

Reply:
Sorry. My ignorance getting in the way there.

Reply:
f044018d05a7aab93e00b1e85a06daaaafad243c